GCRCatalogs/dc2_coadd.py

```diff
@@ -133,38 +131,45 @@ def _generate_modifiers(pixel_scale=0.2):
             'base_PixelFlags_flag_clipped',
         )
 
+        modifiers['good'] = (create_basic_flag_mask,) + not_good_flags
+
+        modifiers['clean'] = (
+            create_basic_flag_mask,
+            'deblend_skipped',
+        ) + not_good_flags
+
         # cross-band average, second moment values
-        modifiers['I_flag'] = 'slot_Shape_flag'
+        modifiers['I_flag'] = 'ext_shapeHSM_HsmSourceMoments'
```

Comment:
@wmwv `ext_shapeHSM_HsmSourceMoments` does not exist in the hdf5 files.

Reply:
maybe you mean `ext_shapeHSM_HsmSourceMoments_flag`?

Reply:
Yes, thank you.

Reply:
OK, fixed.